### DISCUSSION TITLE
Snooze alert menu UI problems since migrating from Pixel 5 to 8a

### DISCUSSION THREAD

**Discussion by kaussel** [NightscoutFoundation/xDrip#3668] [General]
Hello community,

after migrating from an Android 14 Pixel 5 to an Pixel 8a I encounter two problems with xDrip. One is more cosmetic and already listed as issue in Github (I also have the strange yellow lines on lock screen using wall).

The other one is really annoying. The UI does not show all menu items under "Snooze alert" when no alert is active. The UI then shows only the top of the usual three, e.g. "Disable Low Alerts" (s. screenshot). This box is active and can be used . Below that I can only see and use a fraction of "Disable High Alerts", e.g. if I'm really precise and hit the small stripe that is shown I can use it. I do never see the third "Disable All Alerts" box and therefore cannot hit/activate it by randomly pressing the black background.

![xdrid-ui](https://github.com/user-attachments/assets/6d206293-ed1a-4e5c-8305-de32c58b7e2a)

The annoying part is not being able to snooze alerts in advance knowing that they'll be triggered soon. If an alert condition is met everything is working fine.

Android 14 Build AP2A.240905.003.A1
xDrip Version 27de713-2024.05.59

Anyone else had similar issues or am I the unicorn?

Best regards
Joeran

**Comment by Navid200** [comment]
Update xDrip.  
![Screenshot 2024-09-17 085002](https://github.com/user-attachments/assets/c3500f0f-baa9-441c-a734-7c7c65fe8aae)

**Comment by kaussel** [comment]
Thank you very much. I was on beta channel and not offered newer versions. Switching to alpha fixed it. 

Best regards
